ts/Series/MapBubble/MapBubbleSeries.ts

```diff
@@ -273,7 +275,9 @@ class MapBubbleSeries extends BubbleSeries {
     translate(): void {
         MapPointSeries.prototype.translate.call(this);
         this.getRadii();
+        (this as any).adjustMinSize = true;
         this.translateBubble();
+        (this as any).adjustMinSize = false;
```

Comment:
Here (inside `translateBubble`) I need to fire modified (modified by this `adjustMinSize` flag) `getPxExtremes` method.
Any ideas on how can I do it better?